pkg/monitor/monitor/monitor_manager.go

```diff
@@ -250,12 +249,12 @@ func (m *MonitorManager) syncTidbMonitorConfig(tc *v1alpha1.TidbCluster, dc *v1a
 				continue
 			}
 			selector := labels.NewSelector().Add(*r1).Add(*r2)
-			tcList, err := m.deps.Clientset.PingcapV1alpha1().TidbClusters(tcRef.Namespace).List(metav1.ListOptions{LabelSelector: selector.String()})
+			tcList, err := m.deps.TiDBClusterLister.List(selector)
```

Comment:
Why do not match `tcRef.Namespace` anymore?

Reply:
done